### PR TITLE
 add nomics-test v3 ea

### PR DESCRIPTION
**Pull request by karen-stepanyan**

### CODE REVIEW

File: packages/sources/nomics-test/src/endpoint/filtered.ts
Comment (alejoberardino) [inline packages/sources/nomics-test/src/endpoint/filtered.ts:55]: Seems like nomics supports batches on the base side, but not the quote. Same thing as the kaiko one, we should block this one.

**Review by alejoberardino — DISMISSED** [review]

File: packages/sources/nomics-test/src/config/index.ts
Comment (boxhock) [inline packages/sources/nomics-test/src/config/index.ts:6]: Make this sensitive?

File: packages/sources/nomics-test/src/crypto-utils.ts
Comment (boxhock) [inline packages/sources/nomics-test/src/crypto-utils.ts:55]: Can we extract this into its own type and re-use that? Assuming it's the same payload for each of these time ranges

File: packages/sources/nomics-test/src/endpoint/filtered.ts
Comment (boxhock) [reply packages/sources/nomics-test/src/endpoint/filtered.ts:50]: Can this be unblocked now? Can we make this EA send batches on the base, like the crypto endpoint?

**Review by boxhock — CHANGES_REQUESTED** [review]

File: packages/sources/nomics-test/src/endpoint/filtered.ts
Comment (karen-stepanyan) [reply packages/sources/nomics-test/src/endpoint/filtered.ts:50]: I don't really think `/prices/restricted` API endpoint is partially batchable. As a matter of fact i didn't even find any documentation for that. There is a `prices` API endpoint where just one ticker is allowed.

File: packages/sources/nomics-test/src/config/index.ts
Comment (bogos) [reply packages/sources/nomics-test/src/config/index.ts]: Done

File: packages/sources/nomics-test/src/endpoint/filtered.ts
Comment (boxhock) [reply packages/sources/nomics-test/src/endpoint/filtered.ts:50]: Should we just mark this as resolved then?

**Review by boxhock — APPROVED** [review]
LGTM

**Comment by amit-momin** [comment]
I think this is good to go but let's do one more framework bump before we merge this. The recent patches had some important fixes

**Review by amit-momin — APPROVED** [review]

**Pull request merged**
